closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -1626,6 +1714,59 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_RSAEP
     return (jint)msg_len;
 }
 
+/* RSADP Cryptographic Primitive, RSA Private Key operation
+ * Returns -1 on error
+ * The param verify is -1 for 'no verify', otherwise it is size of m (with verify)
+ *
+ * Class:     jdk_crypto_jniprovider_NativeCrypto
+ * Method:    RSADP
+ * Signature: ([BI[BIJ)I
+ */
+
+typedef struct RSADP_BN {
+    BN_CTX *ctx;
+    BIGNUM *k_bigNum;
+    BIGNUM *m_bigNum;
+    BIGNUM *rem;
+} RSADP_BN;
```

Comment:
The comment seems to be for a function rather than a type declaration; please fix.